### PR TITLE
some minor changes

### PR DESCRIPTION
**Pull request by xeon-zolt**
changed system-lock.py to system-lock
changed env to python3 from python2

### CODE REVIEW

**Comment by meets2tarun** [comment]
@xeon-zolt  Kindly create an issue then refer your PR to that issue.

**Comment by xeon-zolt** [comment]
#7

File: usr/bin/system-lock [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```
Comment (Abhi2424shek) [inline usr/bin/system-lock]: Let the shebang line tell the interpreter to reference `/usr/bin/env`

Comment (xeon-zolt) [inline usr/bin/system-lock]: done

**Pull request merged**